.github/workflows/kind-scaffolding-e2e-createtree.yml

```diff
@@ -0,0 +1,76 @@
+name: Kind scaffolding test
+
+on:
+  pull_request:
+    branches: [ 'main', 'release-*' ]
+
+permissions: read-all
+
+jobs:
+  e2e-tests:
+    name: test tree creation with scaffolding
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false # Keep running if one leg fails.
+      matrix:
+        k8s-version:
+        - v1.22.x
+        - v1.23.x
```

Comment:
Any reason not to add newer kube versions?

Reply:
LOL, oh dear :) Yes, done.